src/views/selectmodel.js

```diff
@@ -29,11 +32,11 @@ exports.initSelectModel = function(ctx) {
           $('<a>').attr('role', 'menu-item')
             .attr('href', '/')
             .attr('data-model', model.name)
-            .append(model.description)
+            .append(model.description.substring(0, model.description.length - 1), model.rate==8000?' (8KHz)':' (16KHz)'))
```

Comment:
yes, that is right, broad and narrowband models for different languages. Sampling rates above 16k get converted to 16k at the service level.